python/nano/test/pytorch/tests/test_inference_optimizer.py

```diff
@@ -125,6 +127,8 @@ def test_pipeline_without_metric(self):
                                thread_num=1)
 
         acc_model, option = inference_opt.get_best_model()
+        if TORCH_VERSION_LESS_1_10:
+            return
```

Comment:
1.9 and there tests has been deprecated in previouse PR, we are good to remove these

Reply:
done